app/src/androidTest/java/org/wikipedia/espresso/travel/DestinationHistoryEspressoTest.java

```diff
@@ -0,0 +1,83 @@
+package org.wikipedia.espresso.travel;
+
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.ViewInteraction;
+import android.support.test.filters.SdkSuppress;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.uiautomator.UiDevice;
+import android.support.test.uiautomator.UiObject;
+import android.support.test.uiautomator.UiObjectNotFoundException;
+import android.support.test.uiautomator.UiSelector;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.wikipedia.R;
+import org.wikipedia.travel.TravelPlannerActivity;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.hasDescendant;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+
+@SdkSuppress(minSdkVersion = 18)
+public class DestinationHistoryEspressoTest {
+    private UiDevice mDevice;
+
+    @Rule
+    public ActivityTestRule<TravelPlannerActivity> dActivityTestRule = new ActivityTestRule<>(TravelPlannerActivity.class);
+
+    @Before
+    public void setUp() {
+        //Allows testing of components outside of testing context, in this case: Google Android Places Autocomplete
+        mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+
+        //Navigate to the Destinations fragment
+        ViewInteraction planNewTripButton = onView(
+                allOf(withId(R.id.trip_button_new)));
+        planNewTripButton.perform(click());
+    }
+
+    @Test
+    public void selectedDestinationShouldAppearInDestinationHistory() throws UiObjectNotFoundException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)